webapp/views/App/views/Users/UserEdit/UserEdit.js

```diff
@@ -38,79 +39,100 @@ const UserEdit = () => {
-            disabled={!canEditName}
-            placeholder={canEditName ? i18n.t('common.name') : i18n.t('usersView.notAcceptedYet')}
-            value={User.getName(userToUpdate)}
-            validation={canEditName ? Validation.getFieldValidation(User.keys.name)(validation) : {}}
-            maxLength={User.nameMaxLength}
-            onChange={(value) => onUpdate({ name: User.keys.name, value })}
-          />
-        </FormItem>
-        <FormItem label={i18n.t('common.email')}>
-          <Input
-            disabled={!canEditEmail}
-            placeholder={i18n.t('common.email')}
-            value={User.getEmail(userToUpdate)}
-            validation={Validation.getFieldValidation(User.keys.email)(validation)}
-            onChange={(value) => onUpdate({ name: User.keys.email, value })}
-          />
-        </FormItem>
-        <FormItem label={i18n.t('common.group')}>
-          <DropdownUserGroup
-            editingLoggedUser={User.isEqual(user)(userToUpdate)}
-            disabled={!canEditGroup}
-            validation={Validation.getFieldValidation(User.keys.groupUuid)(validation)}
-            groupUuid={User.getGroupUuid(userToUpdate)}
-            onChange={(value) => onUpdate({ name: User.keys.groupUuid, value })}
-          />
-        </FormItem>
-
-        {(canEdit || User.isInvitationExpired(userToUpdate)) && (
-          <div className="user-edit__buttons">
-            {canRemove && (
-              <button type="button" className="btn-s btn-danger btn-remove-user" onClick={onRemove}>
-                <span className="icon icon-bin icon-left icon-10px" />
-                {i18n.t('userView.removeFromSurvey')}
-              </button>
-            )}
-
-            {canEdit && (
-              <button
-                type="button"
-                className="btn btn-save"
-                aria-disabled={!Validation.isValid(validation)}
-                onClick={onSave}
-              >
-                <span className="icon icon-floppy-disk icon-left icon-12px" />
-                {i18n.t('common.save')}
-              </button>
-            )}
-
-            {User.isInvitationExpired(userToUpdate) && (
-              <button type="button" className="btn btn-invite" onClick={onInviteRepeat}>
-                <span className="icon icon-envelop icon-left icon-12px" />
-                {i18n.t('userView.sendNewInvitation')}
-              </button>
-            )}
-          </div>
-        )}
-      </div>
-    )
+    <div className="user-edit form">
+      {canEdit ? (
+        <ProfilePictureEditor
+          userUuid={userUuid}
+          onPictureUpdate={(profilePicture) => onUpdateProfilePicture({ profilePicture })}
+          enabled
+        />
+      ) : (
+        <ProfilePicture userUuid={userUuid} />
+      )}
+
+      <FormItem label={i18n.t('common.title')}>
+        <Dropdown
+          onChange={onUpdateTitle}
+          items={titleItems}
+          selection={User.Props.getTitle(userToUpdate)}
```

Comment:
we never use Object.Props.
Add a method called User.getTitle. 
The fact that title is a props, is hidden to the consumer of the object.

Reply:
Thanks, Agree!

should we change this 
`User.assocPropsTitle(value)(currentProps)``
to
User.assocTitle and hide the implementation to the user into the User model?